log_based_simulation_ws/src/person_detection_log_based_simulation/package.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>person_detection_log_based_simulation</name>
+  <version>1.0.0</version>
+  <description>
+    AWS RoboMaker log based simulation package for demonstrating Rekognition by detecting faces.
+  </description>
+  <license>MIT</license>
+  <author email="ros-contributions@amazon.com">AWS RoboMaker</author>
+  <maintainer email="ros-contributions@amazon.com">AWS RoboMaker</maintainer>
+  <buildtool_depend>catkin</buildtool_depend>
```

Comment:
just confirming that we do not need `aws-robomaker-simulation-ros-pkgs` because we are telling customers to run `rosws update` ? do we have plan to get `aws-robomaker-simulation-ros-pkgs` into apt store?

Reply:
note: need to flip `aws-robomaker-simulation-ros-pkgs` to public

Reply:
already made public, resolving this review

Reply:
> just confirming that we do not need `aws-robomaker-simulation-ros-pkgs` because we are telling customers to run `rosws update` ? do we have plan to get `aws-robomaker-simulation-ros-pkgs` into apt store?

We do plan to bloom aws-robomaker-simulation-ros-pkgs.  I suggest leaving it here for now. It does no harm and makes explicit the 3P dependency.